packages/wallets/strike/src/adapter.ts

```diff
@@ -0,0 +1,181 @@
+
+    private _connecting: boolean;
+    private _wallet: StrikeWallet | null;
+    private _publicKey: PublicKey | null;
+    private _readyState: WalletReadyState =
+        typeof window === 'undefined' ? WalletReadyState.Unsupported : WalletReadyState.Loadable;
+
+
+    constructor(config: StrikeWalletAdapterConfig = {}) {
+        super();
+        this._connecting = false;
+        this._wallet = null;
+        this._publicKey = null;
+
+        if (this._readyState !== WalletReadyState.Unsupported) {
+            scopePollingDetectionStrategy(() => {
+                if (this._wallet) {
+                    this._readyState = WalletReadyState.Loadable;
+                    this.emit('readyStateChange', this._readyState);
+                    return true;
+                }
+                return false;
+            });
+        }
```

Comment:
Check the Torus adapter, this code isn't needed for Loadable wallets at all.

Reply:
yup, got it, thanks